user-lock-manager-w-keypad-support.smartapp.groovy

```diff
@@ -48,6 +50,7 @@ def rootPage() {
         input name: "maxUsers", title: "Number of users", type: "number", multiple: false, refreshAfterSelection: true, submitOnChange: true
         href(name: "toSetupPage", title: "User Settings", page: "setupPage", description: setupPageDescription(), state: setupPageDescription() ? "complete" : "")
         href(name: "toInfoPage", page: "infoPage", title: "Lock Info")
+        href(name: "toKeypadPage", page: "keypadPage", title: "Keypad Info (optional)")
```

Comment:
Wrap this in an if block too..
if (keypads) {
  href(name: "toKeypadPage", page: "keypadPage", title: "Keypad Info (optional)")
}